### PR TITLE
fix: replace "anatomy" links and remove folder

### LINKED ISSUE #101: links to "anatomy/" still in a lot of places
Hi, it seems the `anatomy/` has been renamed into `building/`. I noticed it by arriving on a 404 from a link in another issue. It seems there are still many places referencing the old name and using old links like in the readme of this repo for example.

### PR DESCRIPTION
**Pull request by kvalium**
FIxes https://github.com/exercism/docs/issues/101

* Replace `anatomy` link by `building` on `README.md`
* Remove `anatomy` folder

### CODE REVIEW

**Review by verdammelt — APPROVED** [review]

File: anatomy/track-tooling/analyzers [hunk 1]
```diff
@@ -1 +0,0 @@
-This has moved to https://github.com/exercism/docs/tree/main/building/tooling/analyzers
```
Comment (ErikSchierboom) [inline anatomy/track-tooling/analyzers]: Please don't remove this file, as it is there to gracefully handle old links.

File: anatomy/tracks/config-json.md [hunk 2]
```diff
@@ -1 +0,0 @@
-This file has moved [here](https://github.com/exercism/docs/blob/main/building/tracks/config-json.md)
```
Comment (ErikSchierboom) [inline anatomy/tracks/config-json.md]: Please don't remove this file, as it is there to gracefully handle old links.

File: anatomy/tracks/configlet/linting.md [hunk 3]
```diff
@@ -1 +0,0 @@
-This file has moved to https://github.com/exercism/docs/blob/main/building/configlet/lint.md
```
Comment (ErikSchierboom) [inline anatomy/tracks/configlet/linting.md]: Please don't remove this file, as it is there to gracefully handle old links.

**Review by ErikSchierboom — COMMENTED** [review]
Thanks for the PR! I've left some comments on not removing the anatomy files.

File: anatomy/track-tooling/analyzers (hunk 1 above)
Comment (kvalium) [reply anatomy/track-tooling/analyzers]: Hello @ErikSchierboom ! I've revert this file removal (and the 2 others as well)

File: anatomy/tracks/config-json.md (hunk 2 above)
Comment (kvalium) [reply anatomy/tracks/config-json.md]: Removal has been reverted

File: anatomy/tracks/configlet/linting.md (hunk 3 above)
Comment (kvalium) [reply anatomy/tracks/configlet/linting.md]: Removal has been reverted

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Thanks @kvalium!

**Pull request merged**